// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

syntax = "proto2";

message t_phr_stats {
   required uint32 du_ue_index = 1;
   required uint32 serv_cell_id = 2;
   required uint32 ph_min = 3;
   required uint32 ph_max = 4;
   required uint32 p_cmax_min = 5;
   required uint32 p_cmax_max = 6;
}

message phr_stats {
   required uint64 timestamp = 1;
   repeated t_phr_stats stats = 2;
}





